prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -0,0 +1,66 @@
+"""Redfish collector."""
+
+from logging import getLogger
+from typing import Dict, List
+
+from ..utils import Command
+
+logger = getLogger(__name__)
+
+
+class RedfishSensors(Command):
+    """Command line tool for getting sensor data using redfish."""
+
+    prefix = ""
+    command = "rf_sensor_list.py"
```

Comment:
**issue:** 

Where is this file?

Reply:
it's in https://github.com/DMTF/Redfish-Tacklebox/tree/main/scripts

Reply:
Additional note: When we install the `redfish-utilities` library, we can use this as a command directly (as it's in `/usr/local/bin`).